### ISSUE TITLE
Building UIExplorer on Windows

### ISSUE THREAD

**Issue by leocavalcante** [facebook/react-native#5788]
Guys,
I'm stuck on this:
```
D:/android/ndk/toolchains/arm-linux-androideabi-4.8/prebuilt/windows-x86_64/bin/arm-linux-androideabi-g++ -MMD -MP -MF W:\react-native\ReactAndroid\build\tmp\buildReactNdkLib/local/armeabi-v7a/objs/reactnative/JSCWebWorker.o.d -fpic -ffunction-sections -funwind-tables -fstack-protector -no-canonical-prefixes -march=armv7-a -mfpu=vfpv3-d16 -mfloat-abi=softfp -fno-exceptions -fno-rtti -mthumb -Os -g -DNDEBUG -fomit-frame-pointer -fno-strict-aliasing -finline-limit=64 -IW:/react-native/ReactAndroid/src/main/jn                                                               i//react/.. -IW:/react-native/ReactAndroid/src/main/jni/first-party/fb/.. -IW:/react-native/ReactAndroid/src/main/jni/first-party/fb/include -IW:/react-native/ReactAndroid/src/main/jni/first-party/jni/.. -IW:/react-native/ReactAndroid/build/third-party-ndk/folly -IW:/react-native/ReactAndroid/build/third-party-ndk/jsc -ID:/android/ndk/sources/cxx-stl/gnu-libstdc++/4.8/include -ID:/android/ndk/sources/cxx-stl/gnu-libstdc++/4.8/libs/armeabi-v7a/include -ID:/android/ndk/sources/cxx-stl/gnu-libstdc++/4.8/include/backward -IW:/react-native/ReactAndroid/build/third-party-ndk/boost/boost_1_57_0 -IW:/react-native/ReactAndroid/build/third-party-ndk/glog/.. -IW:/react-native/ReactAndroid/build/third-party-ndk/glog/glog-0.3.3/src/ -IW:/react-native/ReactAndroid/build/third-party-ndk/double-conversion -IW:/react-native/ReactAndroid/src/main/jni//react -DANDROID -DLOG_TAG=\"ReactNative\" -Wall -Werror -std=c++11 -Wa,--noexecstack -Wformat -Werror=format-security -std=c++11 -std=gnu++11 -std=gnu++11 -DFOLLY_NO_CONFIG=1 -DFOLLY_HAVE_CLOCK_GETTIME=1 -std=gnu++11 -std=c++11 -Wno-unused-variable -Wno-unused-local-typedefs -frtti -fexceptions     -ID:/android/ndk/platforms/android-9/arch-arm/usr/include -c  W:/react-native/ReactAndroid/src/main/jni//react/JSCWebWorker.cpp -o W:\react-native\ReactAndroid\build\tmp\buildReactNdkLib/local/armeabi-v7a/objs/reactnative/JSCWebWorker.o
make.exe: Leaving directory `W:/react-native/ReactAndroid/src/main/jni/react/jni'
:ReactAndroid:buildReactNdkLib FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':ReactAndroid:buildReactNdkLib'.
> Process 'command 'D:\android\ndk\ndk-build.cmd'' finished with non-zero exit value 2

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 22.729 secs
```

Any ideias?

**Comment by leocavalcante** [comment]
Additional notes:

I'm able to run
```bash
D:/android/ndk/toolchains/arm-linux-androideabi-4.8/prebuilt/windows-x86_64/bin/arm-linux-androideabi-g++ -MMD -MP -MF W:\react-native\ReactAndroid\build\tmp\buildReactNdkLib/local/armeabi-v7a/objs/reactnative/JSCWebWorker.o.d -fpic -ffunction-sections -funwind-tables -fstack-protector -no-canonical-prefixes -march=armv7-a -mfpu=vfpv3-d16 -mfloat-abi=softfp -fno-exceptions -fno-rtti -mthumb -Os -g -DNDEBUG -fomit-frame-pointer -fno-strict-aliasing -finline-limit=64 -IW:/react-native/ReactAndroid/src/main/jn                                                               i//react/.. -IW:/react-native/ReactAndroid/src/main/jni/first-party/fb/.. -IW:/react-native/ReactAndroid/src/main/jni/first-party/fb/include -IW:/react-native/ReactAndroid/src/main/jni/first-party/jni/.. -IW:/react-native/ReactAndroid/build/third-party-ndk/folly -IW:/react-native/ReactAndroid/build/third-party-ndk/jsc -ID:/android/ndk/sources/cxx-stl/gnu-libstdc++/4.8/include -ID:/android/ndk/sources/cxx-stl/gnu-libstdc++/4.8/libs/armeabi-v7a/include -ID:/android/ndk/sources/cxx-stl/gnu-libstdc++/4.8/include/backward -IW:/react-native/ReactAndroid/build/third-party-ndk/boost/boost_1_57_0 -IW:/react-native/ReactAndroid/build/third-party-ndk/glog/.. -IW:/react-native/ReactAndroid/build/third-party-ndk/glog/glog-0.3.3/src/ -IW:/react-native/ReactAndroid/build/third-party-ndk/double-conversion -IW:/react-native/ReactAndroid/src/main/jni//react -DANDROID -DLOG_TAG=\"ReactNative\" -Wall -Werror -std=c++11 -Wa,--noexecstack -Wformat -Werror=format-security -std=c++11 -std=gnu++11 -std=gnu++11 -DFOLLY_NO_CONFIG=1 -DFOLLY_HAVE_CLOCK_GETTIME=1 -std=gnu++11 -std=c++11 -Wno-unused-variable -Wno-unused-local-typedefs -frtti -fexceptions     -ID:/android/ndk/platforms/android-9/arch-arm/usr/include -c  W:/react-native/ReactAndroid/src/main/jni//react/JSCWebWorker.cpp -o W:\react-native\ReactAndroid\build\tmp\buildReactNdkLib/local/armeabi-v7a/objs/reactnative/JSCWebWorker.o
```
And compile JSCWebWorker.cpp individually:

![IMG](http://s7.postimg.org/478ht79ln/built.jpg)

The error maybe is with something that should run after this?

**Comment by janicduplessis** [comment]
Are you on master? The build seems broken at the moment since fc94f1e6d0ac61e250c13d293ff76c9c5a5dbbf5 you can try to checkout the commit before and see if it works.

**Comment by leocavalcante** [comment]
Thank you @janicduplessis!
Just checked out 0.20-stable, worked.

**Comment by efkan** [comment]
@leocavalcante , how to build UIExplorer on windows? 
I cannot..

**Comment by leocavalcante** [comment]
Hey @efkan, I can't help you without knowing what exactly is going wrong with your try.
What have you tried and what are the logs/error messages?

**Comment by efkan** [comment]
You're right. But I just want to know what steps you did.

I've tried lots of ways. But I couldn't manage..

One of methods that official steps;

```
git clone https://github.com/facebook/react-native.git
cd react-native
npm install
```

However I encountered with this issue; 
http://stackoverflow.com/questions/34986728/react-native-example-uiexplorer-building-issue

Now desperately I'm trying to implement to a new project after `react-native init ...`.
